pkgs/by-name/pi/piano-rs/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  pkg-config,
+  alsa-lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "piano-rs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ritiek";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-qZeH9xXQPIOJ87mvLahnJB3DuEgLX0EAXPvECgxNlq0=";
+  };
+
+  cargoHash = "sha256-vDqfWXeQVEnMWMjhAG/A0afff7dWMoQejDZjcVlYBMQ=";
+
+  nativeBuildInputs = [
+    makeWrapper
```

`makeBinaryWrapper` is slightly faster to start.

```suggestion
    makeBinaryWrapper
```